content/tutorials/single-page-tutorials/hornet-inx-interaction.md

```diff
@@ -0,0 +1,313 @@
+# INX Interface Interaction
+
+In this tutorial, you will learn how to interact with the IOTA Node Extension ([INX](https://github.com/iotaledger/inx)) Interface via javascript (NodeJS) with both simple requests and streaming requests. The extension uses the [inx.proto](https://github.com/iotaledger/inx/blob/develop/proto/inx.proto) file to generate the required gRPC client to connect to HORNET. [gRPC](https://grpc.io/) is a modern open source high performance Remote Procedure Call (RPC) framework that can run in any environment.

```

Comment:
```suggestion
In this tutorial, you will learn how to interact with the IOTA Node Extension ([INX](https://github.com/iotaledger/inx)) interface via JavaScript (NodeJS) with both simple requests and streaming requests. The extension uses the [inx.proto](https://github.com/iotaledger/inx/blob/develop/proto/inx.proto) file to generate the required gRPC client to connect to HORNET. [gRPC](https://grpc.io/) is a modern open source high performance Remote Procedure Call (RPC) framework that can run in any environment.
```

"interface" is not part of the name and should not be capitalized. "JavaScript" is a name and should be capitalized by its creator's spelling.